### PR TITLE
MLDB-1927 import text named new default

### PR DESCRIPTION
**Pull request by FinchPowers**
From
```
lineNumber()
```

To
```
dataFileUrl() + '.' + lineNumber()
```

### CODE REVIEW

File: plugins/importtext_procedure.cc [hunk 1]
```diff
@@ -77,7 +77,7 @@ ImportTextConfigDescription::ImportTextConfigDescription()
     addField("named", &ImportTextConfig::named,
              "Row name expression for output dataset. Note that each row "
              "must have a unique name.",
-             SqlExpression::parse("lineNumber()"));
+             SqlExpression::parse("dataFileUrl() + '.' + lineNumber()"));
```
Comment (jeremybarnes) [inline plugins/importtext_procedure.cc]: Should be [ dataFileUrl(), lineNumber() ]

Comment (FinchPowers) [inline plugins/importtext_procedure.cc]: I'm not clear on your suggestion. My 2 attemps don't work.
* `dataFileUrl(), lineNumber()` fails with `dataFileUrl(), lineNumber():1:14 ('dataFileUrl()>>>,<<< lineNumber('): expected eof`
* `[dataFileUrl(), lineNumber()]` fails with `Cannot convert value of type 'embedding' to 'atomic value' (Utf8 string)`

Why isn't the way I did it ok?

Comment (simlmx) [inline plugins/importtext_procedure.cc]: Faut que tu créees un ColumnPath ou qqch du genre non?

Comment (guyd) [inline plugins/importtext_procedure.cc]: File path can be very long.  I am concerned that a fair portion of the imported dataset memory footprint will be comprised of the repeated file path.  It seems to me that this is a problem specific to the merge dataset.  Why not change the merge dataset to prefix/suffix the row names of the datasets?  Something like`[]-[12]`for rows in the right dataset and `[729]-[]` for rows in the left dataset.

Comment (FinchPowers) [inline plugins/importtext_procedure.cc]: Suite de la discussion dans le billet.

**Comment by FinchPowers** [comment]
Scrappé en faveur d'une meilleure solution.

**Pull request closed without merging**